drivers/storage/portworx/util/util.go

```diff
@@ -247,6 +249,10 @@ const (
 	// VsphereInstallModeLocal env value for Vsphere 'local' install
 	VsphereInstallModeLocal = "local"
 )
+const (
+	// pxEntriesKey is key which holds all the bootstrap entries
+	pxEntriesKey = "px-entries"
```

Comment:
Can you completely remove the duplicate `pxEntriesKey` constant from status.go and replace all occurrences with this one?

Reply:
Updated